docs/docs/reference/config-migration.md

```diff
@@ -1,8 +1,14 @@
-# Configuration migrations
+# Migrations
 
-This document describes breaking changes in the configuration file format between Constellation releases.
+This document describes breaking changes and migrations between Constellation releases.
 Use [`constellation config migrate`](./cli.md#constellation-config-migrate) to automatically update an old config file to a new format.
 
+## Migrating from Azure's service principal authentication to managed identity authentication
+
+- The `provider.azure.appClientID` and `provider.azure.appClientSecret` fields are no longer required and should be removed.
+- To keep using an existing UAMI add the `Owner` permission with the scope of your `resourceGroup`.
+- Otherwise, simply [create new Constellation IAM credentials](../workflows/config.md#creating-iam-credentials) and use the created UAMI.
+
```

Comment:
I am not super convinced of my suggestion below so feel free to not follow it:

We could also explain how to patch the `azureconfig` k8s secret to use the uami (and no longer depend on the app id / app secret).

Reply:
Ah yes, I actually just forgot about it. I think we should at least point to the secret name and the docs of the `cloud-provider-azure`. But since it can be done with a single command, we can also just add that.